manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ObjectReference.java

```diff
@@ -0,0 +1,62 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+public class ObjectReference {
```

Comment:
Same comment for this and [`BaseResponse`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/infra/src/main/java/com/redhat/service/smartevents/infra/api/models/responses/BaseResponse.java)